src/test/java/uk/gov/pay/products/infra/DropwizardAppWithPostgresRule.java

```diff
@@ -70,7 +70,7 @@ public void evaluate() throws Throwable {
                 restoreDropwizardsLogging();
 
                 DataSourceFactory dataSourceFactory = app.getConfiguration().getDataSourceFactory();
```

Comment:
Is it still needed?

Reply:
Good spot! Not required any more and removed